definitions/ext-ddmysql/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: DATADOG_MYSQL
+
+synthesis:
+  name: dd.mysql
+  identifier: dd.mysql
+  encodeIdentifierInGUID: false
```

Comment:
If `dd.mysql` values are the equivalents of a hostname this should be true

Reply:
I ended up realizing I didn't actually understand what these are derived from, so it looks like the value will actually be `host.name` based on what we got from the metric.  Switching the boolean to true as well.